common/src/java/com/zimbra/common/util/Version.java

```diff
@@ -177,10 +177,6 @@ public int compareTo(Version version) {
         if (r != 0)
             return r;
 
-        r = mPatch - version.mPatch;
```

Comment:
please update unit tests also zm-mailbox/common/src/java-test/com/zimbra/common/util/VersionTest.java

Reply:
Updated the VersionTest.java according to current scenario. I will add new test cases too.

Reply:
The method changed is used at these places where it has been actively used in zimbra desktop and Zimbra/zm-touch-client repo· https://github.com/search?q=org%3AZimbra%20Version.compare(&type=code
The method will be unused in that case if these repos are not being actively used

Reply:
touch client is only present in 8815, and this fix will be part of Zimbra 10.x only
about zimbra desktop you are referring to deprecated version of zimbra desktop, which we don't use or release anymore

Reply:
Done. Thanks